src/modules/dashboard/sagas/extensions.ts

```diff
@@ -368,6 +371,48 @@ function* colonyExtensionUpgrade({
   return null;
 }
 
+function* updateWhitelist({
+  meta,
+  payload: { userAddress, colonyAddress, status },
+}: Action<ActionTypes.UPDATE_WHITELIST>) {
+  const txChannel = yield call(getTxChannel, meta.id);
+  const apolloClient = TEMP_getContext(ContextModule.ApolloClient);
+  // @TODO Please extend this saga to fit batch update when adding addresses to the list
```

Comment:
Also note that when doing batching, they need to split up into multiple transactions, not just one